video/out/mac/title_bar.swift

```diff
@@ -221,7 +221,6 @@ class TitleBar: NSVisualEffectView {
         default:                break
         }
 
-#if HAVE_MACOS_10_11_FEATURES
         if #available(macOS 10.11, *) {
```

Comment:
This this `if #available(macOS 10.11, *) {` still needed too? I mean it doesn't harm to keep it, but if you are breaking compatibility, might as well here too.

Reply:
Heh, I just commented on that. I would assume probably not but I don't know swift or use mac so I figured I'd just let someone else that actually knows it handle that.